src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -5691,6 +5728,7 @@ static void directionJump(XmlWriter& xml, const Jump* const jp)
         xml.startElement("direction", { { "placement", (jp->placement() == PlacementV::BELOW) ? "below" : "above" } });
         xml.startElement("direction-type");
         String positioning = ExportMusicXml::positioningAttributes(jp);
+        positioning += color2xml(jp);
```

Comment:
please move before `positioning`, as in most other cases

Reply:
There's a bit of inconsistence, sometimes color is last
Anyway, done